org.sugarlabs.MusicBlocks.json

```diff
@@ -24,7 +24,7 @@
                     "type": "git",
                     "url": "https://github.com/sugarlabs/musicblocks-launcher",
                     "tag": "v1.0.4",
```

Comment:
Read my comment above, and don't forget to bump your tag.

Reply:
Just changed it!